internal/fields/validate.go

```diff
@@ -323,7 +320,7 @@ func initDependencyManagement(packageRoot string, specVersion semver.Version, im
 		if err != nil {
 			return nil, nil, err
 		}
-		logger.Debug("Imported ECS fields definition from external schema for validation")
+		logger.Debug("Imported ECS fields definition from external schema for validation (package: %v, stack: %v)", packageEmbedsEcsMappings, stackSupportsEcsMapping)
```

Comment:
Nit. Requires `Debugf` and `packageEmbedsECSMappings` is a bool

This is the current message shown ([link](https://buildkite.com/elastic/elastic-package/builds/2824#018ee696-716e-4961-9127-5cd9a74690e6/111-938)):
```
Imported ECS fields definition from external schema for validation (package: %v, stack: %v)false true
```

Maybe too descriptive like this?

```suggestion
		logger.Debugf("Imported ECS fields definition from external schema for validation (package embeds ECS: %v, stack supports ECS: %v)", packageEmbedsEcsMappings, stackSupportsEcsMapping)
```